### ISSUE TITLE
How to convert Trained Model(Backbone : Faster RCNN) to ONNX and How to deploy a inference environment??

### ISSUE THREAD

**Issue by JeongHanJun** [facebookresearch/unbiased-teacher-v2#13]
Hello, thank you for such a great study. 
What I'm curious about is that Unbased Teacher v2 is how to convert the learned Faster RCNN model based on Detectron2 to ONNX. 
I referred to the contents of this repository and train the model (Faster RCNN) based on COCO-Detection/faster_rcnn_R_101_FPN_3x config. 
Now I want to convert it to ONNX. 
I installed Detectron2 based on Docker, so I'd appreciate your help on how to deploy the inference environment and the model converted to ONNX!

In short,
How can I convert a learned model to ONNX?
and How to execute the inference of the learned model regardless of the docker container?

I would be very grateful if you could help me with this.
